### ISSUE TITLE
Upgrading v4 to v5 not working

### ISSUE THREAD

**Issue by Mynty100** [grokability/snipe-it#9142]
#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
Unable to upgrade to v5 from v4.9.5.

**To Reproduce**
running php upgrade.php

**Screenshots**
![image](https://user-images.githubusercontent.com/34461276/108018386-52e8e500-7063-11eb-98f6-4538a417384f.png)

**Server (please complete the following information):**
 - Snipe-IT Version = v4.9.5
 - OS: Ubuntu
 - Web Server: Apache
 - PHP Version: 7.3

Sever has been updating through all releases of SNIPEIT v4. We have simple waiting for v5 to be a bit more mature and tested upgrading today. Ran into issues with composer and updated that manually fixed some issues but now we stack here.
[SnipeIT upgrade errors.txt](https://github.com/snipe/snipe-it/files/5986079/SnipeIT.upgrade.errors.txt)
I have attached a full log of the install taking place and failing.

Thanking you,
Mynty.

**Comment by Mynty100** [comment]
I have also tried changing the php version to v7.2.
I have tried running the following
$ ./vendor/bin/upgrade-carbon
git pull origin
git reset --hard
Currently running Composer v2.0.9
No luck so far.

**Comment by Mynty100** [comment]
Resolved.
Upgraded OS to newer version as technically we running Linux Mint 18 with is based on Ubuntu. Originally had to trick snipeIT install to thinking the OS was Ubuntu. Now on Mint 20.1.

After this we then got the .env file error found that fix  Fixed the .env file and done.
Also been having mkdir error on backups but fixed this as well with help from.
https://github.com/snipe/snipe-it/issues/7172
sudo chown -R www-data:www-data ./storage
sudo chmod -R 777 ./storage 
(had to set 777 for backups to work during upgrade)

Everything now seems to work.
